internal/resolvers/imgur/load_test.go

```diff
@@ -0,0 +1,365 @@
+						{
+							Title:       "My Cool Title",
+							Description: "My Cool Description",
+							Datetime:    int(datetime),
+						},
+					},
+				},
+				Limit: &imgur.RateLimit{},
+			}, 420, nil)
+
+		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> Album Title</li><li><b>Description:</b> Album Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li></div>`
+
+		cleanTooltip := testLoadAndUnescape(c, url)
+
+		c.Assert(cleanTooltip, qt.Equals, expectedTooltip)
+	})
+
+	c.Run("Gallery HTML tags album", func(c *qt.C) {
+		const url = "empty_gallery_album"
+		m.
+			EXPECT().
+			GetInfoFromURL(gomock.Eq(url)).
+			Return(&imgur.GenericInfo{
+				Image:  nil,
+				Album:  nil,
+				GImage: nil,
+				GAlbum: &imgur.GalleryAlbumInfo{
+					Title:       "Album <b>Title</b>",
+					Description: "Album <b>Description</b>",
+					DateTime:    int(datetime),
+					ImagesCount: 1,
+					Images: []imgur.ImageInfo{
+						{
+							Title:       "My Cool Title",
+							Description: "My Cool Description",
+							Datetime:    int(datetime),
+						},
+					},
+				},
+				Limit: &imgur.RateLimit{},
+			}, 420, nil)
+
+		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> Album &lt;b&gt;Title&lt;/b&gt;</li><li><b>Description:</b> Album &lt;b&gt;Description&lt;/b&gt;</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li></div>`
+
+		cleanTooltip := testLoadAndUnescape(c, url)
+
+		c.Assert(cleanTooltip, qt.Equals, expectedTooltip)
+	})
+
+	// TODO: Convert to Gallery images
+	c.Run("Gallery image", func(c *qt.C) {
+		const url = "gallery_image"
+		m.
+			EXPECT().
+			GetInfoFromURL(gomock.Eq(url)).
+			Return(&imgur.GenericInfo{
+				Image: nil,
+				Album: nil,
+				GImage: &imgur.GalleryImageInfo{
+					Title:       "My Cool Title",
+					Description: "My Cool Description",
+					Datetime:    int(datetime),
+				},
+				GAlbum: nil,
+				Limit:  &imgur.RateLimit{},
+			}, 420, nil)
+
+		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> My Cool Title</li><li><b>Description:</b> My Cool Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li></div>`
+
+		cleanTooltip := testLoadAndUnescape(c, url)
+
+		c.Assert(cleanTooltip, qt.Equals, expectedTooltip)
+	})
+
+	c.Run("NSFW gallery image", func(c *qt.C) {
+		const url = "nsfw_gallery_image"
+		m.
+			EXPECT().
+			GetInfoFromURL(gomock.Eq(url)).
+			Return(&imgur.GenericInfo{
+				Image: nil,
+				Album: nil,
+				GImage: &imgur.GalleryImageInfo{
+					Title:       "My Cool Title",
+					Description: "My Cool Description",
+					Datetime:    int(datetime),
+					Nsfw:        true,
+				},
+				GAlbum: nil,
+				Limit:  &imgur.RateLimit{},
+			}, 420, nil)
+
+		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> My Cool Title</li><li><b>Description:</b> My Cool Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li><li><b><span style="color: red">NSFW</span></b></li></div>`
```

Comment:
Keep consistency with css semicolons
```suggestion
		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> My Cool Title</li><li><b>Description:</b> My Cool Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li><li><b><span style="color: red;">NSFW</span></b></li></div>`
```